views/js/qtiItem/core/Loader.js

```diff
@@ -414,6 +414,10 @@ define([
         },
         loadPicData : function(pic, data){
             loadPortableCustomElementData(pic, data);
+        },
+
+        stripNamespaces: function (body) {
```

Comment:
Sorry, you may not be aware of that, but the function should be named. I know this is a legacy code, and even if the rest of the file is not following the good practices we are following now, could you at least apply it on this function? The reason why is while in debug session, it is difficult to know in which function we are if it is anonymous.

```
stripNamespaces: function stripNamespaces(body) {
...
}
```

Reply:
Sounds good